plugins/outputs/elasticsearch/README.md

```diff
@@ -0,0 +1,34 @@
+# Elasticsearch Output Plugin
+
+This plugin writes to [Elasticsearch](https://www.elastic.co) via Elastic (http://olivere.github.io/elastic/).
+
+Attention: 
+ Elasticsearch 2.x does not support this dots-to-object transformation and so dots in field names are not allowed in versions 2.X.
+In this case, dots will be replaced with "_".
+
+### Configuration:
+
+```toml
+# Configuration for Elasticsearch to send metrics to
+[[outputs.elasticsearch]]
+  ## The full HTTP endpoint URL for your Elasticsearch. # required
+  server_host = "http://10.10.10.10:9200"
+  ## The target index for metrics # required
```

Comment:
can you explain more about this auto-index-creating system you've come up with? why is it necessary? is it something the general user would need? Is there a precedent from another product?

also, if it needs to stay, you need to explain more about the template being used. When do the timestamps get evaluated? when does the index get created?, etc...